src/App.jsx

```diff
@@ -49,7 +49,7 @@ const App = () => {
     dispatch(fetchEmployees());
     dispatch(fetchTimeOff());
     dispatch(fetchRoles());
-    dispatch(fetchView('login'));
+    dispatch(fetchView('employees'));
```

Comment:
Let's get this fixed back to the right login page.

Reply:
😞